IntelliTect.TestTools.TestFramework/ExampleTests/ExampleTests/WebDriverFactory.cs

```diff
@@ -5,7 +5,9 @@
 
 namespace ExampleTests
 {
+#pragma warning disable CA1001 // Types that own disposable fields should be disposable; Driver should be disposed by DI container, not the factory
```

Comment:
Any reason for suppressing this rather than making it disposable?

Reply:
Yeah, it's an odd quirk of this library. Since the factory ultimately returns an IWebDriver (which itself is disposable,) and the container that manages the dependencies should dispose of this on its own at the end of the ExecuteTestCase() method, I don't think making this disposable is the best thing to do here.

However, I think probably a refactor here will address this. I could probably get away with _not_ storing the IWebDriver at all; presumably good isolation between tests removes the need to do this. If you feel strongly this, I can do that refactor for this PR.

Reply:
Refactored to remove suppression.